tests-functional/clients/services/accounts.py

```diff
@@ -0,0 +1,22 @@
+from clients.rpc import RpcClient
+from clients.services.service import Service
+
+
+class AccountService(Service):
+    def __init__(self, client: RpcClient):
+        super().__init__(client, "accounts")
+
+    def get_accounts(self):
+        response = self.rpc_request("getAccounts")
+        return response.json()
+
+    def get_account_keypairs(self):
+        response = self.rpc_request("getKeypairs")
+        return response.json()
+
+    def get_pubkey(self, display_name):
```

Comment:
I'd call it explicitly `find_`. 
```suggestion
    def find_pubkey(self, display_name):
```

And maybe add an assert that only one account found? It's technically possible to have multiple accounts with same `display_name`. Or return list of pubkeys.

Reply:
But actually, such function should not exist. 
With the `node.login` signal, you get the `account` and `settings`. You can use it to get `keyUID` and `publicKey` accordingly.

Reply:
Done, thanks, used `node.login` signal to extract it